pkg/providers/instancetype/suite_test.go

```diff
@@ -243,10 +243,11 @@ var _ = Describe("InstanceTypeProvider", func() {
 			v1.LabelInstanceGPUCount:                     "1",
 			v1.LabelInstanceGPUMemory:                    "16384",
 			v1.LabelInstanceLocalNVME:                    "900",
-			v1.LabelInstanceAcceleratorName:              "inferentia",
-			v1.LabelInstanceAcceleratorManufacturer:      "aws",
-			v1.LabelInstanceAcceleratorCount:             "1",
-			v1.LabelTopologyZoneID:                       "tstz1-1a",
+			// TODO - NVIDIA/GPU instances should not have Neuron/accelerator labels
```

Comment:
Is this TODO still outstanding?

Reply:
yes, that is correct. I noticed this when updating the tests but I have not delved into that to further investigate so left it as a follow up ToDo